internal/schema/v1/advertiser_id.go

```diff
@@ -0,0 +1,28 @@
+package schema
+
+import "github.com/Decentr-net/cerberus/internal/schema/types"
+
+// AdvertiserID is id for advertiser..
+type AdvertiserID struct {
+	Advertiser string `json:"advertiser"`
+	ID         string `json:"id"`
+}
+
+// Type ...
+func (AdvertiserID) Type() types.Type {
+	return types.PDVAdvertiserIDType
+}
+
+// Validate ...
+func (d AdvertiserID) Validate() bool {
+	if d.Advertiser == "" || d.ID == "" {
```

Comment:
Any length validation? Could be any span here?

Reply:
done